code/modules/projectiles/projectile/bullets.dm

```diff
@@ -8,100 +8,233 @@
+	name = "10mm incendiary bullet"
+	damage = 15
+	fire_stacks = 2
+
+// 9mm (Stechkin APS)
+
+/obj/item/projectile/bullet/c9mm
+	name = "9mm bullet"
 	damage = 20
 
-/obj/item/projectile/bullet/toxinbullet
+/obj/item/projectile/bullet/c9mm_ap
+	name = "9mm armor-piercing bullet"
 	damage = 15
-	damage_type = TOX
+	armour_penetration = 40
 
-/obj/item/projectile/bullet/incendiary/firebullet
+/obj/item/projectile/bullet/incendiary/c9mm
+	name = "9mm incendiary bullet"
 	damage = 10
+	fire_stacks = 1
+
+// 4.6x30mm (Autorifles)
+
+/obj/item/projectile/bullet/c46x30mm
+	desc = "4.6x30mm bullet"
+	damage = 20
 
-/obj/item/projectile/bullet/armourpiercing
+/obj/item/projectile/bullet/c46x30mm_ap
+	name = "4.6x30mm armor-piercing bullet"
 	damage = 15
 	armour_penetration = 40
 
-/obj/item/projectile/bullet/pellet
-	name = "pellet"
-	damage = 12.5
+/obj/item/projectile/bullet/incendiary/c46x30mm
+	name = "4.6x30mm incendiary bullet"
+	damage = 10
+	fire_stacks = 1
 
-/obj/item/projectile/bullet/pellet/Range()
-	..()
-	damage += -0.75
-	if(damage < 0)
-		qdel(src)
+// .45 (M1911)
 
-/obj/item/projectile/bullet/pellet/weak
-	damage = 6
+/obj/item/projectile/bullet/c45
+	name = ".45 bullet"
+	damage = 20
+	stamina = 65
 
-/obj/item/projectile/bullet/pellet/weak/New()
-	range = rand(1, 8)
-	..()
+/obj/item/projectile/bullet/c45_nostamina
+	name = ".45 bullet"
+	damage = 20
 
-/obj/item/projectile/bullet/pellet/weak/on_range()
-	do_sparks(1, TRUE, src)
-	..()
+// 5.56mm (M-90gl Carbine)
 
-/obj/item/projectile/bullet/pellet/overload
-	damage = 3
+/obj/item/projectile/bullet/a556
+	name = "5.56mm bullet"
+	damage = 35
 
-/obj/item/projectile/bullet/pellet/overload/New()
-	range = rand(1, 10)
-	..()
+// 40mm (Grenade Launcher
 
-/obj/item/projectile/bullet/pellet/overload/on_hit(atom/target, blocked = FALSE)
- 	..()
- 	explosion(target, 0, 0, 2)
+/obj/item/projectile/bullet/a40mm
+	name ="40mm grenade"
+	desc = "USE A WEEL GUN"
+	icon_state= "bolter"
+	damage = 60
 
-/obj/item/projectile/bullet/pellet/overload/on_range()
-	explosion(src, 0, 0, 2)
-	do_sparks(3, TRUE, src)
+/obj/item/projectile/bullet/a40mm/on_hit(atom/target, blocked = FALSE)
 	..()
+	explosion(target, -1, 0, 2, 1, 0, flame_range = 3)
+	return 1	
 
-/obj/item/projectile/bullet/midbullet
-	damage = 20
-	stamina = 65 //two round bursts from the c20r knocks people down
+// .50 (Sniper)
 
+/obj/item/projectile/bullet/p50
+	name =".50 bullet"
+	speed = 0		//360 alwaysscope.
+	damage = 70
+	knockdown = 100
+	dismemberment = 50
+	armour_penetration = 50
+	var/breakthings = TRUE
 
-/obj/item/projectile/bullet/midbullet2
-	damage = 25
+/obj/item/projectile/bullet/p50/on_hit(atom/target, blocked = FALSE)
+	if((blocked != 100) && (!ismob(target) && breakthings))
+		target.ex_act(rand(1,2))
+	return ..()
 
-/obj/item/projectile/bullet/midbullet3
-	damage = 30
+/obj/item/projectile/bullet/p50/soporific
+	name =".50 soporific bullet"
+	armour_penetration = 0
+	nodamage = 1
+	dismemberment = 0

```

Comment:
FALSE

Reply:
Which are you talking about? `armour_penetration` an `dismemberment` are integers, I think.